docs/autoplay.md

```diff
@@ -0,0 +1,82 @@
+## Managing Auto Play
+If you would like to start automatically playing your content, you can easily manipulating the player configuration and set the `playback.autoplay` value accordingly.
+
+>### playback.autoplay
+>##### Type: `boolean`
+>##### Default: `false`
+>##### Description: Start playback automatically.
+>If `true`, playback will start automatically when sources will be given to the player. If `false`, a user gesture will be required to start playback.
+#### Example - Basic usage:
+```js
+var config = {
+  playback: {
+    autoplay: true
+  },
+  sources: {...}
+};
+// No need to add additional code, 
+// playback will start automatically!
+var player = Playkit.loadPlayer(config);
+```
+
+>**_Note_**:
+> Some developers may 
+> simulate the autoplay behavior by calling `play()` method right after the creation of the player without manipulating the player configuration. We strongly recommend **not** doing it this way!
+>```js
```

Comment:
Some developers simulate the autoplay behavior by calling the `play()` method right after creating the Player without manipulating the player configuration. However, we strongly recommend **not** using this method!

Reply:
@AnnaAleksandrowicz 
Maybe my sentence wasn't clear so u didn't understood me.
The thing is that to use this method is fine and necessary - for example if you pause the player, the only way to resume playback is to call play.
What I meant here is not to use it as "fake autoplay" - we want developers to let us manage to autoplay internally in the player core.